SharedPackages/BrowserServicesKit/Sources/NetworkProtection/Routing/VPNRoutingRange.swift

```diff
@@ -46,38 +46,54 @@ public enum VPNRoutingRange {
+        "8.0.0.0/7",        /* 8.0.0.0 - 9.255.255.255 */
+        /* 10.0.0.0/8           10.0.0.0 - 10.255.255.255       Private network */
+        "11.0.0.0/8",       /* 11.0.0.0 - 11.255.255.255 */
+        "12.0.0.0/6",       /* 12.0.0.0 - 15.255.255.255 */
+        "16.0.0.0/4",       /* 16.0.0.0 - 31.255.255.255 */
+        "32.0.0.0/3",       /* 32.0.0.0 - 63.255.255.255 */
+        "64.0.0.0/2",       /* 64.0.0.0 - 127.255.255.255 */
+        /* 127.0.0.0            127.0.0.0/8 - 127.255.255.255   Loopback */
+        "128.0.0.0/3",      /* 128.0.0.0 - 159.255.255.255 */
+        "160.0.0.0/5",      /* 160.0.0.0 - 167.255.255.255 */
+        "168.0.0.0/8",      /* 168.0.0.0 - 168.255.255.255 */
+        "169.0.0.0/9",      /* 169.0.0.0 - 169.127.255.255 */
+        "169.128.0.0/10",   /* 169.128.0.0 - 169.191.255.255 */
+        "169.192.0.0/11",   /* 169.192.0.0 - 169.223.255.255 */
+        "169.224.0.0/12",   /* 169.224.0.0 - 169.239.255.255 */
+        "169.240.0.0/13",   /* 169.240.0.0 - 169.247.255.255 */
+        "169.248.0.0/14",   /* 169.248.0.0 - 169.251.255.255 */
+        "169.252.0.0/15",   /* 169.252.0.0 - 169.253.255.255 */
+        /* 169.254.0.0/16       169.254.0.0 - 169.254.255.255   Link-local */
+        "169.255.0.0/16",   /* 169.255.0.0 - 169.255.255.255 */
+        "170.0.0.0/7",      /* 170.0.0.0 - 171.255.255.255 */
+        "172.0.0.0/12",     /* 172.0.0.0 - 172.15.255.255 */
+        /* 172.16.0.0/12        172.16.0.0 - 172.31.255.255     Private network */
+        "172.32.0.0/11",    /* 172.32.0.0 - 172.63.255.255 */
+        "172.64.0.0/10",    /* 172.64.0.0 - 172.127.255.255 */
+        "172.128.0.0/9",    /* 172.128.0.0 - 172.255.255.255 */
+        "173.0.0.0/8",      /* 173.0.0.0 - 173.255.255.255 */
+        "174.0.0.0/7",      /* 174.0.0.0 - 175.255.255.255 */
+        "176.0.0.0/4",      /* 176.0.0.0 - 191.255.255.255 */
+        "192.0.0.0/9",      /* 192.0.0.0 - 192.127.255.255 */
+        "192.128.0.0/11",   /* 192.128.0.0 - 192.159.255.255 */
+        "192.160.0.0/13",   /* 192.160.0.0 - 192.167.255.255 */
+        /* 192.168.0.0/16       192.168.0.0 - 192.168.255.255   Private network*/
+        "192.169.0.0/16",   /* 192.169.0.0 - 192.169.255.255 */
+        "192.170.0.0/15",   /* 192.170.0.0 - 192.171.255.255 */
+        "192.172.0.0/14",   /* 192.172.0.0 - 192.175.255.255 */
+        "192.176.0.0/12",   /* 192.176.0.0 - 192.191.255.255 */
+        "192.192.0.0/10",   /* 192.192.0.0 - 192.255.255.255 */
+        "193.0.0.0/8",      /* 193.0.0.0 - 193.255.255.255 */
+        "194.0.0.0/7",      /* 194.0.0.0 - 195.255.255.255 */
+        "196.0.0.0/6",      /* 196.0.0.0 - 199.255.255.255 */
+        "200.0.0.0/5",      /* 200.0.0.0 - 207.255.255.255 */
+        "208.0.0.0/4",      /* 208.0.0.0 - 223.255.255.255 */
+        /* 224.0.0.0            224.0.0.0/4 - 239.255.255.255   Multicast */
+        /* 240.0.0.0            240.0.0.0/4 - 255.255.255.255   Class E */
```

Comment:
Diff sucks as it is, but I added these comments to make the ranges easier to track visually.

The only change should be that 168.0.0.0/6 is replaced by subranges not including link-local.